### PR TITLE
Update zulu8.rb

### PR DESCRIPTION
**Pull request by robertleeblairjr**
Update check version for Intel and Arm architecture builds.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/zulu8.rb [hunk 1]
```diff
@@ -1,29 +1,36 @@
...
     end
   end
 
+  name "Azul Zulu Java 8 Standard Edition Development Kit"
+  desc "OpenJDK distribution from Azul"
+  homepage "https://www.azul.com/downloads/"
+  
   depends_on macos: ">= :yosemite"
 
   pkg "Double-Click to Install Zulu #{version.major}.pkg"
```
Comment (bevanjkay) [inline Casks/zulu8.rb:29]: The pkg has changed path. 

```suggestion
  pkg "Double-Click to Install Azul Zulu JDK #{version.major}.pkg"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/zulu8.rb [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
+      strategy :page_match do |page|
+        match = page.match(%r{url"\s*:\s*"https:.*?/zulu(\d+(?:\.\d+)*-.*?)-jdk(\d+(?:\.\d+)*)-macos}i)
+        "#{match[2]},#{match[1]}"
+      end
     end
   end
 
+  name "Azul Zulu Java 8 Standard Edition Development Kit"
+  desc "OpenJDK distribution from Azul"
+  homepage "https://www.azul.com/downloads/"
```
Comment (bevanjkay) [inline Casks/zulu8.rb:32]:
```suggestion
  homepage "https://www.azul.com/products/core/"
```

To match other Zulu versioned casks.

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/zulu8.rb (hunk 1 above)
Comment (robertleeblairjr) [reply Casks/zulu8.rb:29]: The other is the common one across the other zulu casks.

File: Casks/zulu8.rb (hunk 2 above)
Comment (robertleeblairjr) [reply Casks/zulu8.rb]: I changed to the main homepage.

File: Casks/zulu8.rb (hunk 1 above)
Comment (bevanjkay) [reply Casks/zulu8.rb:29]: @robertleeblairjr The other casks have been updated to the new file path with version bumps in the last 24 hours. It was flagged in one of the earlier audit failures.

Comment (bevanjkay) [reply Casks/zulu8.rb:29]: @robertleeblairjr 
From the CI run
> Error: Could not find PKG source file 'Double-Click to Install Zulu 8.pkg', found 'Double-Click to Install Azul Zulu JDK 8.pkg' instead.

Comment (robertleeblairjr) [reply Casks/zulu8.rb:29]: I see what you're talking about after I checked the error. I'll change it.

**Review by bevanjkay — APPROVED** [review]
@robertleeblairjr Thanks for your hard work on this one 🎉

**Pull request merged**